base/src/main/java/uk/ac/ebi/atlas/experimentimport/ExperimentCrud.java

```diff
@@ -89,11 +84,6 @@ public UUID importExperiment(String experimentAccession, boolean isPrivate) thro
             deleteExperiment(experimentAccession);
         }
 
-        // We only insert in the DB differential and single cell experiments expressions
-        analyticsLoaderFactory
```

Comment:
Does data prod have data loading scripts for Expression Atlas yet...?

Reply:
No, not yet. From the webapp we only need to load coexpressions and index in Solr.